### PR TITLE
[BCM] set brcmpcmsink low_latency property for webrtc streams

### PR DESCRIPTION
**Pull request by goruklu**
Signed-off-by: Gurdal Oruklu <gurdal_oruklu@comcast.com>

### CODE REVIEW

**Review by modeveci — APPROVED** [review]
Looks good to me. @eocanha it is a platform specific change. can you also look at that?

File: Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamer.cpp [hunk 1]
```diff
@@ -3891,6 +3891,15 @@ void MediaPlayerPrivateGStreamer::configureElement(GstElement* element)
     if (g_str_has_prefix(GST_ELEMENT_NAME(element), "brcmaudiosink")) {
         g_object_set(G_OBJECT(element), "async", TRUE, nullptr);
     }
+    if(!g_strcmp0(G_OBJECT_TYPE_NAME(G_OBJECT(element)), "GstBrcmPCMSink")) {
+#if ENABLE(MEDIA_STREAM)
+        if (player->m_streamPrivate != nullptr && g_object_class_find_property(G_OBJECT_GET_CLASS(element), "low_latency")) {
```
Comment (eocanha) [inline Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamer.cpp]: This code is inside a class member function, where you can directly use `m_streamPrivate` (no need for `player->`). In any case, the `player` variable wasn't defined, so this line shouldn't compile without the suggested correction.
Take advantage of this change to also remove `!= nullptr` and simply use `m_streamPrivate`. On C++ that's enough for not null checking.
UPDATE: Oh, I see that you already got rid of `player` in a follow-up patch in this same PR. Then the whole patch is good enough to be merged.

File: Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamer.cpp [hunk 2]
```diff
@@ -3891,6 +3891,15 @@ void MediaPlayerPrivateGStreamer::configureElement(GstElement* element)
...
+            GST_DEBUG("Set 'low_latency' in brcmpcmsink");
+            g_object_set (G_OBJECT(element), "low_latency", TRUE, nullptr);
```
Comment (eocanha) [inline Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamer.cpp:3898]: If you feel like and also think that this makes things simple, you can set the two properties at the same time with a single `g_object_set(object, "property1", value1, "property2", value2, ..., nullptr)` call. Do it or not at your own discretion.

**Review by eocanha — CHANGES_REQUESTED** [review]
Appart from the provided comments, everything looks fine.

**Review by eocanha — APPROVED** [review]
After a second read, the PR is fine enough to be squashed and merged.

**Pull request merged**